products/sle/main.pm

```diff
@@ -686,6 +690,7 @@ sub load_inst_tests {
             loadtest "installation/user_settings";
         }
         loadtest "installation/user_settings_root";
+
```

Comment:
please don't. I understand everyone has a different style preference but we should not change back and forth single lines